* Install to omniidl_be outside PYTHONPATH
* Don't overwrite omnidl_be/__init__.py
* Do not blindly recompile files that belong to middleware/omniORB

--- python2/omniidl_be/dir.mk~	2003-03-23 22:51:59.000000000 +0100
+++ python2/omniidl_be/dir.mk	2024-06-20 16:50:23.972847507 +0200
@@ -1,12 +1,12 @@
 PYLIBROOT= $(EXPORT_TREE)/lib/python
 PYLIBDIR = $(PYLIBROOT)/omniidl_be
-INSTALLPYLIBDIR = $(INSTALLPYTHONDIR)/omniidl_be
+INSTALLPYLIBDIR = $(INSTALLLIBDIR)/omniidl/omniidl_be
 
 ifeq ($(PYTHON),)
 PYTHON = python
 endif
 
-FILES = __init__.py python.py
+FILES = python.py
 
 export:: $(FILES)
 	@(dir="$(PYLIBDIR)"; \
@@ -14,7 +14,5 @@
             $(ExportFileToDir) \
           done; \
-          cd $(PYLIBDIR); \
-	  $(PYTHON) -c "import compileall; compileall.compile_dir('.')"; \
 	 )
 
 ifdef INSTALLTARGET
@@ -24,6 +23,4 @@
             $(ExportFileToDir) \
           done; \
-          cd $(INSTALLPYLIBDIR); \
-	  $(PYTHON) -c "import compileall; compileall.compile_dir('.')"; \
 	 )
 endif
